include/swift/Option/Options.td

```diff
@@ -904,4 +904,10 @@ def disable_autolinking_runtime_compatibility : Flag<["-"], "disable-autolinking
   Flags<[FrontendOption]>,
   HelpText<"Do not use autolinking for runtime compatibility libraries">;
 
+def disable_autolinking_runtime_compatibility_dynamic_replacements
+    : Flag<[ "-" ],
+    "disable-autolinking-runtime-compatibility-dynamic-replacements">,
+      Flags<[ FrontendOption ]>,
+      HelpText<"Do not use autolinking for runtime compatibility libraries">;
```

Comment:
Please fix the formatting here to match the other lines, and please change the help text.

…does this really need a separate flag, though?